packages/react-charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -50,8 +52,7 @@ export class DonutChartBase extends React.Component<IDonutChartProps, IDonutChar
 
     let heightState: { _height: number } | undefined;
     if (nextProps.height && nextProps.height !== prevState._height) {
-      const reducedHeight = nextProps.height / 5;
-      heightState = { _height: nextProps.height - reducedHeight };
+      heightState = { _height: nextProps.height - LEGEND_CONTAINER_HEIGHT };
```

Comment:
Currently how is the arc width decided? We might need to tweak that logic for show labels.

Reply:
![image](https://user-images.githubusercontent.com/110246001/217722147-34fbd419-2db6-413c-ad8f-2a4dc56ff2c5.png)